radio/src/functions.cpp

```diff
@@ -394,12 +398,19 @@ void evalFunctions(const CustomFunctionData * functions, CustomFunctionsContext
             newActiveFunctions |= (1u << FUNCTION_DISABLE_TOUCH);
         	break;
 #endif
+#if defined(COLORLCD)
+          case FUNC_SET_SCREEN:
+            if (isRepeatDelayElapsed(functions, functionsContext, i)) {
+              TRACE("SET VIEW %d", (CFN_PARAM(cfn)));
+              ViewMain::instance()->setCurrentMainView((CFN_PARAM(cfn)));
+            }
+#endif
+            break;
 #if defined(DEBUG)
-          case FUNC_TEST:
-            testFunc();
+                case FUNC_TEST : testFunc();
```

Comment:
the indentation looks strange

Reply:
That (410) and the closing brace (413) can be un-indented which will remove them from the PR as changes...

Reply:
Re-formatted.

Reply:
Thanks! Next time, when it's a predominately formatting only change (and affects a decent chunk of code like this did), can you do it in it's own commit - it makes it easier to separate formatting from the rest of the PR...